woocommerce/woocommerce-android
# Feature/7720 domain selection screen

Closes: #7720 


### Description

Adds layout for domain picking step from store creation. 

Expected design can by found here: HyVloP5FipZzyPVenH2euI-fi-2116%3A106887


For this screen existing UI components and styles where reused so matching between Figma design and Android is not 100% exact: 


### Testing instructions

- Log into the app
- From site picker click on "+ Add a store" button
- Click on continue to skip "Questions" step
- Land on Domain picker and check the following (keep in mind data is hardcoded for now): 
  - Typing any input in the search box shows an `X` button. Clicking on it, will clear the search box
  - Clicking on any domain suggestion will mark the domain as selected
  - Scrolling works as expected 
  - Clicking on back button navigates to previous step

**Known issue:** Design does not work in landscape. Opened discussion here: p1667819207157209/1667809684.884529-slack-C045CUK1Y3U

https://user-images.githubusercontent.com/2663464/200303233-8b132b16-6c0d-4944-b1bb-f93b92821661.mp4
